frontend/src/assets/icons.tsx

```diff
@@ -367,3 +367,39 @@ export const MultiSelectTagIcon = (): React.ReactElement => (
     />
   </Icon>
 );
+
+export const TextOutlinedIcon = (): React.ReactElement => (
+  <Icon viewBox="0 0 24 24">
+    <rect width="24" height="24" opacity="0" />
+    <path fill="currentColor" d="M15 16H9a1 1 0 0 0 0 2h6a1 1 0 0 0 0-2z" />
+    <path fill="currentColor" d="M9 14h3a1 1 0 0 0 0-2H9a1 1 0 0 0 0 2z" />
+    <path
+      fill="currentColor"
+      d="M19.74 8.33l-5.44-6a1 1 0 0 0-.74-.33h-7A2.53 2.53 0 0 0 4 4.5v15A2.53 2.53 0 0 0 6.56 22h10.88A2.53 2.53 0 0 0 20 19.5V9a1 1 0 0 0-.26-.67zM14 5l2.74 3h-2a.79.79 0 0 1-.74-.85zm3.44 15H6.56a.53.53 0 0 1-.56-.5v-15a.53.53 0 0 1 .56-.5H12v3.15A2.79 2.79 0 0 0 14.71 10H18v9.5a.53.53 0 0 1-.56.5z"
+    />
+  </Icon>
+);
+
+export const SaveOutlinedIcon = (): React.ReactElement => (
+  <Icon viewBox="0 0 24 24">
+    <rect width="24" height="24" opacity="0" />
+    <path
+      fill="currentColor"
+      d="M20.12 8.71l-4.83-4.83A3 3 0 0 0 13.17 3H6a3 3 0 0 0-3 3v12a3 3 0 0 0 3 3h12a3 3 0 0 0 3-3v-7.17a3 3 0 0 0-.88-2.12zM10 19v-2h4v2zm9-1a1 1 0 0 1-1 1h-2v-3a1 1 0 0 0-1-1H9a1 1 0 0 0-1 1v3H6a1 1 0 0 1-1-1V6a1 1 0 0 1 1-1h2v5a1 1 0 0 0 1 1h4a1 1 0 0 0 0-2h-3V5h3.17a1.05 1.05 0 0 1 .71.29l4.83 4.83a1 1 0 0 1 .29.71z"
+    />
+  </Icon>
+);
+
+export const EyeOutlinedIcon = (): React.ReactElement => (
```

Comment:
nit: `EyeOutlinedIcon` -> `EyeOutlineIcon` to be consistent with the naming above 
same for `TextOutlinedIcon` and `SaveOutlinedIcon`.